src/components/SelectionListSingle/Footer.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import type {GestureResponderEvent, StyleProp, ViewStyle} from 'react-native';
+import Button from '@components/Button';
+import FixedFooter from '@components/FixedFooter';
+import useThemeStyles from '@hooks/useThemeStyles';
+
+type ConfirmButtonProps = {
+    showConfirmButton?: boolean;
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    onConfirm?: (e?: GestureResponderEvent | KeyboardEvent | undefined, option?: any) => void;
+    confirmButtonStyle?: StyleProp<ViewStyle>;
+    confirmButtonText?: string;
+    isConfirmButtonDisabled?: boolean;
+};
+
+type FooterProps = {
+    footerContent?: React.ReactNode;
+    confirmButton?: ConfirmButtonProps;
```

Comment:
Minor thing but is seems like `confirmButtonConfig` would be a better fit for the name - initailly I thought it would be a react component